### PR TITLE
Addition of k8s-apply plugin

### PR DESCRIPTION
**Pull request by daniel-de-vera**
Check the [README](https://github.com/signadot/plugins/tree/k8s-apply/k8s-apply) for details.

### CODE REVIEW

**Comment by foxish** [comment]
Could you please also add a note to the docs with a reference to this plugin? Thanks!

**Comment by daniel-de-vera** [comment]
> LGTM - had already reviewed once in hackspace and I assume this is the same. Thanks!

Yes, code-wise it is the same (only some package reorg), but here I added the example and README.

**Comment by foxish** [comment]
README lgtm

File: k8s-apply/src/Makefile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+DOCKER ?= docker
+
+build-docker:
+	$(DOCKER) build -t signadot/k8s-apply:latest .
```
Comment (scott-cotton) [inline k8s-apply/src/Makefile:4]: Do we want to version the image?

File: k8s-apply/src/cmd/k8s_apply/main.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	}
+	mapper := restmapper.NewDeferredDiscoveryRESTMapper(memory.NewMemCacheClient(dc))
+
+	// prepare the dynamic client
+	dyn, err := dynamic.NewForConfig(cfg)
+	if err != nil {
+		panic(err.Error())
+	}
+
+	ctx := context.Background()
```
Comment (scott-cotton) [inline k8s-apply/src/cmd/k8s_apply/main.go:50]: Maybe a timeout would be appropriate for this ctx, since it is doing queries.

**Review by scott-cotton — APPROVED** [review]
LGTM,

I would suggest we version this, so we can change :latest without any surprises for consumers

File: k8s-apply/src/cmd/k8s_apply/main.go (hunk 2 above)
Comment (daniel-de-vera) [reply k8s-apply/src/cmd/k8s_apply/main.go]: It's done here: https://github.com/signadot/plugins/pull/19/commits/78a29d8005f3c32f3af0f63308f2863e7e2dc592.

File: k8s-apply/src/Makefile (hunk 1 above)
Comment (daniel-de-vera) [reply k8s-apply/src/Makefile]: It's done here: https://github.com/signadot/plugins/pull/19/commits/78a29d8005f3c32f3af0f63308f2863e7e2dc592.

**Pull request merged**